src/m365/planner/commands/task/task-add.ts

```diff
@@ -126,10 +138,20 @@ class PlannerTaskAddCommand extends GraphCommand {
       });
   }
 
+  private generateAppliedCategories(options: Options): AppliedCategories {
+    if (!options.appliedCategories) {
+      return {};
+    }
+
+    const categories: AppliedCategories = {};
+    this.splitByCommaOrSpace(options.appliedCategories).forEach(x => categories[x] = true);
```

Comment:
In the spec and it docs we say that we expect a comma-separated list. Why do we allow using space as a separator as well?

Reply:
Split doesn't work in PowerShell if the user runs the command as `--option variable1,variable2` (without the quotes ""). @martinlingstuyl noticed this here https://github.com/pnp/cli-microsoft365/pull/3315#discussion_r876957550. Because this occurs in multiple other commands, the following issue was created #3340

Reply:
Won't we have the same issue in PowerShell if we use space as a separator rather than a comma, without surrounding quotes? And wouldn't adding quotes to a comma-separated string solve the issue with split in PowerShell?

Reply:
Maybe it's a good idea to remove the split by space functionality here for the time being, @Jwaegebaert, @waldekmastykarz. We can followup on that discussion in the other issue and merge this pull request without it.